README.md

```diff
@@ -54,7 +54,7 @@ Hello there,<br>
 I am Scotty (Aka. Scottywonderful), I hope this finds you well and that you are having a great day.<br>
 I am pleased to announce that this is my first ever callout pack created for LSDPFR.<br>
 With this, I am excited to say that I have multiple ideas and updates for this pack in the future.<br>
-That's all from me, have fun, remember to think safe, act safe, be safe!<br>
+That's all from me, have fun, remember to think safe, act safe, and be safe!<br>
```

Comment:
This is a good motto from a transportation company I use to do work with :) So I won't change this :)

Reply:
I have decided to leave the and in the edit. Thanks for the feedback.